app/grandchallenge/cases/models.py

```diff
@@ -145,22 +103,18 @@ def process_images(self, linked_task=None):
             status=RawImageUploadSession.REQUEUED
         )
 
-        kwargs = {"args": (self.pk,)}
+        kwargs = {"upload_session_pk": self.pk}
+        workflow = build_images.signature(kwargs=kwargs)
 
         if linked_task is not None:
-            kwargs.update(
-                {
-                    "link": linked_task.signature(
-                        kwargs={"upload_session_pk": self.pk}
-                    )
-                }
-            )
+            linked_task.kwargs.update(kwargs)
```

Comment:
I _think_ that it means that they cannot be updated once the canvas has been sent to the broker (otherwise the arguments are updated with the results of the previous tasks), but I could be wrong. It does indeed work, and there is a test to check, but it would be good to know for sure :-)